data/docker-nginx.conf

```diff
@@ -0,0 +1,57 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+
+    server {
+
+        set $root_path '/var/www/html/src';
+        listen 80;
+
+        server_name localhost;
```

Comment:
Better is this was the default server using 'server_name _;' as this is only being used for the one application.

Reply:
If I understood it correctly, done.